terraform/Azure/only_private_blob_storage/only_private_blob_storage_test.rego

```diff
@@ -0,0 +1,13 @@
+package torque
+
+test_allow_private_blob_storages {
+  result:= deny with input as data.plan_mock
+  count(result) == 0
+}
+
+test_validate_deny_message {
+  result:= deny with input as data.plan_mock_2
+  expected_deny_message:= "Deployment of not private blob storage is not allowed"
```

Comment:
Same, please fix the error message according to Segev's design.

Reply:
fixed